src/main/java/wolox/gmjavatraining/models/Book.java

```diff
@@ -0,0 +1,44 @@
+package wolox.gmjavatraining.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+import lombok.Data;
+
+@Entity
+@Data
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
```

Comment:
Knowing that we are using Postgres as the database, we should use the SEQUENCE strategy which uses independently auto increment ids for each table.
The AUTO strategy for Postgres is to have a global auto increment, so you first user in your database will have the id 1. But if after that you create your first book, it will have the id 2, because the id 1 was taken from users